spec/octokit/client/pages_spec.rb

```diff
@@ -5,56 +5,93 @@
   before do
     Octokit.reset!
     @client = oauth_client
+    @pages_repo =  "#{test_github_login}/#{test_github_login}.github.io"
```

Comment:
Could we make/ensure this repo exists before the tests start? Maybe in an RSpec context?